k8s/demo/common/ccd/definition-store-api.yaml

```diff
@@ -19,12 +19,24 @@ spec:
   values:
     java:
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-PASS-V11
+            - ccd-definition-s2s-secret
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
```

Comment:
do you know if you need to redefine all the secrets?
Or can we omit `AppInsightsInstrumentationKey, ccd-definition-s2s-secret, storage-account-primary-connection-string` and those will be picked up by default from `values.yml`?

Reply:
I had checked that , it does not apply this logic for vault secrets.  given that secrets is the  key map value  we need to overwrite everything.    It looks likes it is like that.